### PR TITLE
Video message

### PR DESCRIPTION
**Pull request by godoakbrutal**

### CODE REVIEW

File: src/Template/VideoTemplate.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+{
+    /**
+     * @var string
+     */
+    public $videoUrl;
+
+    /**
+     * @var string
+     */
+    public $videoPosterUrl;
```
Comment (liverbool) [inline src/Template/VideoTemplate.php]: i think just `url` no duplicate word  `video`

`VideoTemplate::videoUrl`
`VideoTemplate::url`
`VideoTemplate::posterUrl`

Comment (godoakbrutal) [reply src/Template/VideoTemplate.php]: Done

**Comment by liverbool** [comment]
@godoakbrutal 👍 good work
-- please change

`ImageTemplate::previewImageUrl`
to
`ImageTemplate::previewUrl`

**Comment by godoakbrutal** [comment]
@liverbool 

Done kub.

File: src/Template/ImageTemplate.php [hunk 2]
```diff
@@ -14,13 +14,13 @@ class ImageTemplate extends AbstractTemplate
...
-    public $previewImageUrl;
+    public $previewUrl;
 
     /**
      * {@inheritdoc}
      */
     public function getTemplate()
     {
-        return new ImageMessageBuilder($this->originalUrl, $this->previewImageUrl);
+        return new ImageMessageBuilder($this->originalUrl, $this->previewUrl);
```
Comment (liverbool) [inline src/Template/ImageTemplate.php]: `originalUrl to url` wdyt?

Comment (godoakbrutal) [reply src/Template/ImageTemplate.php]: Ah I agree.

**Pull request merged**